packages/frontend/tsconfig.json

```diff
@@ -0,0 +1,36 @@
+{
+  "compilerOptions": {
+    "composite": false,
+    "declaration": true,
+    "declarationMap": true,
+    "esModuleInterop": true,
+    "forceConsistentCasingInFileNames": true,
+    "inlineSources": false,
+    "isolatedModules": true,
+    "lib": ["dom", "es2020"],
+    "module": "ESNext",
+    "moduleResolution": "node",
+    "noUnusedLocals": false,
+    "noUnusedParameters": false,
+    "outDir": "./dist",
+    "preserveWatchOutput": true,
+    "rootDir": "./src",
+    "skipLibCheck": true,
+    "strict": false,
+    "allowJs": true,
+    "checkJs": true,
```

Comment:
Lets turn this off, and start easy: by just writing new code in TS
Living in a codebase with 100+ files highlighted red is impossible